src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowSnykCodeScanListenerLS.kt

```diff
@@ -40,8 +40,10 @@ class SnykToolWindowSnykCodeScanListenerLS(
 
     override fun scanningSnykCodeFinished(snykCodeResults: Map<SnykCodeFile, List<ScanIssue>>) {
         ApplicationManager.getApplication().invokeLater {
+            this.snykToolWindowPanel.navigateToSourceEnabled = false
             displaySnykCodeResults(snykCodeResults)
             refreshAnnotationsForOpenFiles(project)
+            this.snykToolWindowPanel.navigateToSourceEnabled = true
```

Comment:
I guess this kind of toggling is safe if this function is not called multiple times, different threads situation?